analytics/terraform/datahub-on-eks/datahub-eks/values/datahub_values.yaml

```diff
@@ -0,0 +1,334 @@
+# Values to start up datahub after starting up the datahub-prerequisites chart with "prerequisites" release name
+# Copy this chart and change configuration as needed.
+datahub-gms:
+  enabled: true
+  image:
+    repository: linkedin/datahub-gms
+    tag: "v0.8.45"
+
+datahub-frontend:
+  enabled: true
+  image:
+    repository: linkedin/datahub-frontend-react
+    tag: "v0.8.45"
```

Comment:
Same as above comment. Don't hardcode the tag.

Reply:
removed tags from individual services but still set the 'global' version to 0.8.45, as there seemed to be issue with the latest 0.10.x version working with OpenSearch.   Will remove the version line from the value file once the issue is resolved.